common/app/views/support/HtmlCleaner.scala

```diff
@@ -834,7 +834,13 @@ case class CommercialComponentHigh(isPaidContent: Boolean, isNetworkFront: Boole
     if (containers.length >= minContainers) {
 
       val containerIndex = if (containers.length >= 4) {
-        if (isNetworkFront) 3 else 2
+        if (isNetworkFront) {
```

Comment:
Do we want to check if edition is UK as well in here? I'm not sure if the campaign will run globally or not maybe worth clarifying as different editions have different layout and the switch is global

Reply:
Thanks for the catch. Have updated